cmake/cpu.cmake

```diff
@@ -28,13 +28,13 @@ if (ARM_TARGET AND ARM_TARGET GREATER 6)
         set(XMRIG_ARMv8 ON)
         add_definitions(/DXMRIG_ARMv8)
 
-        CHECK_CXX_COMPILER_FLAG(-march=armv8-a+crypto XMRIG_ARM_CRYPTO)
+        CHECK_CXX_COMPILER_FLAG(-march=armv8-a+crypto+crc XMRIG_ARM_CRYPTO)
```

Comment:
None of the supported algorithms use crc32 so there's no point in adding it to `-march`. Also, the way this change is done, it will fall back to `armv8-a+crc` if `armv8-a+crypto+crc` flag is not available - this will disable AES instructions even if they are available.

Reply:
ok may do specific change for the Mac M1 then

Reply:
But there is no point in adding this at all. XMRig doesn't use CRC32.

Reply:
ah you re right